### PR TITLE
Script for 'Image to Pencil Sketch' Conversion Added

### LINKED ISSUE #1050: Script to convert Image to Pencil Sketch
# Description

The script can be used to convert a RGB image to pencil sketch in 4 easy steps.

## Language

- ✅ Python

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Image to Pencil Sketch | Convert RGB image to pencil sketch. | GSSoC'21 python | @Manasi2001 |

### PR DESCRIPTION
**Pull request by Manasi2001**
# Description

Added the script for converting a RGB image to a pencil sketch.

Fixes #1050 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Image_Sketch/README.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-#### Original Image
-<img src ="https://github.com/sharur7/Rotten-Scripts/blob/sharur7/Python/Image_Sketch/obama.jpg?raw=true" alt="Original_image" width="280" height="300">
-
-#### Sketched Image
-<img src ="https://github.com/sharur7/Rotten-Scripts/blob/sharur7/Python/Image_Sketch/sketch.jpg?raw=true" alt="Original_image" width="280" height="300">
+# CONVERTING IMAGE TO PENCIL SKETCH
+
+### A RGB image can be converted into a pencil sketch using 4 easy steps!
+
+![img.jpg](img.jpg)
```
Comment (vybhav72954) [inline Python/Image_Sketch/README.md:5]: Hey there. Upload all the images on an image hosting system, like imgur. And paste the links here

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: Python/Image_Sketch/README.md (hunk 1 above)
Comment (Manasi2001) [reply Python/Image_Sketch/README.md]: Done. Please check now @vybhav72954.

**Review by vybhav72954 — APPROVED** [review]
The README had multiple issues. I have made an okayish-looking readme. Please next time follow the template

**Comment by Manasi2001** [comment]
> The README had multiple issues. I have made an okayish-looking readme. Please next time follow the template

Got it @vybhav72954. I'll keep that in mind for the next time. Thanks for making the changes! 
Please add the labels too :)

**Pull request merged**